pkg/render/policyrecommendation.go

```diff
@@ -98,6 +99,7 @@ func (pr *policyRecommendationComponent) Objects() ([]client.Object, []client.Ob
 	// Deployment is for standalone or management cluster
 	if !pr.cfg.ManagedCluster {
 		objs = append(objs,
+			pr.externalLinseedRoleBinding(),
```

Comment:
I think you only want the external role binding if this is a managed cluster, but never for management or standalone clusters, __and__ only if there is a component in the managed cluster that needs to talk to Linseed (which IIUC, there isn't for PolicyRecommendation).

The goal of the external role binding is to provide permissions for Linseed to provision access tokens in the PolicyRecommendation namespace in the managed cluster so that components in the managed cluster can hook into RBAC.